sormas-backend/src/test/java/de/symeda/sormas/backend/common/NotificationServiceTest.java

```diff
@@ -58,7 +58,7 @@ public void init() {
 		super.init();
 
 		Mockito.when(configurationFacade.isFeatureEnabled(any())).thenReturn(true);
-		Mockito.when(configurationFacade.isFeatureEnabled(FeatureType.KEYCLOAK_TO_SORMAS_USER_SYNC)).thenReturn(false);
+		Mockito.when(configurationFacade.isFeatureEnabled(FeatureType.AUTH_PROVIDER_TO_SORMAS_USER_SYNC)).thenReturn(false);
```

Comment:
can be deleted

Reply:
the NotificationServiceTests do not pass without this mock. I added it to each failing test.